Gems/ScriptAutomation/Code/Source/ScriptAutomationScriptBindings.cpp

```diff
@@ -269,6 +270,20 @@ namespace ScriptAutomation
             return rpiSystem->GetRenderApiName().GetCStr();
         }
 
+        AZStd::string GetRenderPipelineName()
+        {
+            AZ::IConsole* console = AZ::Interface<AZ::IConsole>::Get();
+            AZStd::string renderPipelinePath;
+            console->GetCvarValue("r_default_pipeline_name", renderPipelinePath);
+            AZ_Assert(renderPipelinePath.size() > 0, "Invalid render pipeline path obtained from r_default_pipeline_name CVAR");
+            return AZ::IO::PathView(renderPipelinePath).Stem().FixedMaxPathString().c_str();
```

Comment:
This can use the `String()` function to return a AZStd::string without converting to a FixedMaxPathString
```suggestion
            AZ::CVarFixedString renderPipelinePath;
            console->GetCvarValue("r_default_pipeline_name", renderPipelinePath);
            AZ_Assert(renderPipelinePath.size() > 0, "Invalid render pipeline path obtained from r_default_pipeline_name CVAR");
            return AZ::IO::PathView(renderPipelinePath).Stem().String();
```